package.json

```diff
@@ -52,6 +52,7 @@
     "es"
   ],
   "devDependencies": {
+    "algoliasearch": "3.30.0",
```

Comment:
We should also add it as a `peerDependency`.

Reply:
You don't have to depend on `algoliasearch` if you want to create your own search client. Should we still list it as a peer dependency?

Reply:
Nope you're right!